Composer/packages/client/package.json

```diff
@@ -55,7 +57,7 @@
     "office-ui-fabric-react": "^7.121.11",
     "prop-types": "^15.7.2",
     "query-string": "^6.8.2",
-    "re-resizable": "^6.3.2",
+    "re-resizable": "^6.9.0",
```

Comment:
Is this a new package? Was there a reason not use the splitter and do the vertical split? Seems like it would do the same function

[@geoffcox/react-splitter - npm (npmjs.com)](https://www.npmjs.com/package/@geoffcox/react-splitter)

Reply:
@GeoffCoxMSFT

Reply:
Actually nvm. Looks like we are using this package in other places. Ignore.